### ISSUE TITLE
Add spice guest tools to the preinstall [Feature Request]

### ISSUE THREAD

**Issue by blitzaxt** [Arc-Compute/LibVF.IO#51]
Without the guest tools installed looking glass does not allow for copying/pasting to/from the host, it would be helpful to have that working from the get-go.

https://www.spice-space.org/download.html

**Comment by arthurrasmusson** [comment]
Upon closer inspection after we talked I think there's a [feature in Looking Glass to support a shared clipboard](https://github.com/gnif/LookingGlass/blob/master/client/src/clipboard.c) using [it's own custom integrated implementation spice](https://github.com/gnif/PureSpice).

**Comment by blitzaxt** [comment]
Yes, and that works but that is the client side as far as I understand. It works as soon as you install the spice guest tools but the clipboard did not seem to work until after I installed the spice guest tools.

**Comment by arthurrasmusson** [comment]
Let me see if I can ask someone to help with this.

**Comment by gnif** [comment]
https://looking-glass.io/docs/B5.0.1/install/#client-clipboard-synchronization

**Comment by blitzaxt** [comment]
That would be awesome @arthurrasmusson! Sorry I have not gotten back, life has been busy.
